uptime_service_validation/coordinator/helper.py

```diff
@@ -294,6 +297,46 @@ def get_existing_nodes(self):
             cursor.close()
         return nodes
 
+    # tuples = [discord_id, block_producer_email, block_producer_key]
+    def update_application_status(self, tuples, page_size=100):
+        "Update the application status of the block producers."
+        self.logger.info("update_application_status  start ")
+        cursor = self.connection.cursor()
+        try:
+            sql = """update nodes set application_status = true, discord_id =%s, email_id =%s
+                where block_producer_key= %s """
+            extras.execute_batch(cursor, sql, tuples, page_size)
+        except (Exception, psycopg2.DatabaseError) as error:
+            self.logger.error(ERROR.format(error))
+            cursor.close()
+            return -1
+        finally:
+            cursor.close()
+            self.connection.commit()
+        self.logger.info("update_application_status  end ")
+        return 0
+
+
+def get_application_details():
```

Comment:
I don't like this name very much. What "application" is it about? Let's call it `get_spreadsheet_connection` or `connect_to_spreadsheet` or something like that.